spec/hashie/array_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe Array do
+  if RUBY_VERSION >= '2.3.0'
```

Comment:
i copied this idiom from [here](https://github.com/modosc/hashie/blob/5488fc618678d7e860e1c63e7830f534a0084cac/lib/hashie/mash.rb#L254) and [here](https://github.com/modosc/hashie/blob/5488fc618678d7e860e1c63e7830f534a0084cac/spec/hashie/mash_spec.rb#L690) - happy to change my code but should i change the other code as well?